cmd/coco/dependencies/graph.go

```diff
@@ -49,7 +50,13 @@ func constructGraph(path, depFileName string) (
 			continue
 		}
 
-		var df depFile
+		df, err := inputfile.Load(p)
+		if !df.IsComponent() {
+			continue
+		}
+		if err != nil {
+			return downToUp, componentPaths, err
+		}
```

Comment:
error must be checked first
```suggestion
		if err != nil {
			return downToUp, componentPaths, err
		}
		if !df.IsComponent() {
			continue
		}
```